plugins/com.google.cloud.tools.eclipse.appengine.facets.test/src/com/google/cloud/tools/eclipse/appengine/facets/ui/AppEngineStandardProjectConvertCommandHandlerTest.java

```diff
@@ -113,6 +115,8 @@ public void testCheckFacetCompatibility_java1_6FacetIsIncompatible() throws Core
 
   @Test
   public void testCheckFacetCompatibility_java1_8FacetIsIncompatible() throws CoreException {
+    Assume.assumeTrue("AppEngine Standard Java 8 runtime support is not present",
```

Comment:
It is a little different. With a feature flag, all code paths are tested regardless of platform. Here the test is sometimes skipped.

More importantly, this raises another question. What does the UI here look like? Are we showing a menu item for the command that does not work in some circumstances? It's just the same existing convert menu item in the context menu, though, right?